src/preprocessing/passes/bv_to_int.cpp

```diff
@@ -688,11 +688,11 @@ Node BVToInt::bvToInt(Node n)
               /*
                * We replace all BV-sorts of the domain with INT
                * If the range is a BV sort, we replace it with INT
-               * We cache both the term itself (e.g., f(a)) and the function
-               * symbol f.
                */
 
-              //Construct the function itself
+              /*
```

Comment:
A minor style comment: I prefer not using `/* */` in comments in cpp (it makes it harder to use block comments for disabling parts of the code for debugging).